cmd/bot.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 var (
-	configPath = flag.String("c", "./config.toml", "config path")
+	configPath = flag.String("c", "./conf", "config path")
```

Comment:
There are some configurations should be shared by all repos, like github token, slack token. I prefer give read config dir path in config file's `include` field.

Reply:
Do you want to support multiple paths in command line parameters?

Reply:
The `include` field here stores the sub configuration list or a single sub configuration?

Reply:
IMO, sub configuration dir.

eg
```toml
[slack]
heartbeat = "admin@pingcap.com"
token = ""
mute = false
hello = false

[github]
token = ""
bot = "cherry-picker"

include = "./repositories"
```

Files:

```
- config.toml
- repositories
  - repo1.toml
  - repo2.toml
  - repo3.toml
```

Reply:
good idea

Reply:
I updated my code, please check. 🐙 ⚡️ 🐱